### ISSUE TITLE
ERROR: Failure: TypeError ('_AtomType' object is not iterable)

### ISSUE THREAD

**Issue by jchodera** [pandegroup/pdbfixer#123]
Travis failure.
```
======================================================================
ERROR: Failure: TypeError ('_AtomType' object is not iterable)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/_test/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/travis/miniconda/envs/_test/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/travis/miniconda/envs/_test/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/travis/miniconda/envs/_test/lib/python2.7/site-packages/pdbfixer/createSoftForcefield.py", line 54, in <module>
    (atomClass, mass, element) = forcefield._atomTypes[atomType]
TypeError: '_AtomType' object is not iterable
```

**Comment by peastman** [comment]
This was fixed in #120.  `createSoftForceField.py` isn't something that should ever get loaded.  It was just a one-off script I wrote to generate the XML file for the soft force field.  But because of the way `nosetests` was being run, it tried to load it as if it were a test case.

It could probably be deleted at this point.  I don't know if there's much value in keeping it around as an historical artifact.

**Comment by jchodera** [comment]
Why close the issue before actually addressing the test failure? Shouldn't we leave this open to execute one of the resolutions you recommend above?  What's the point of automated testing if test failures are ignored?

**Comment by jchodera** [comment]
Huh. Sorry, I hadn't noticed this was fixed in #120. I had thought I had merged in your changes...
